README.md

```diff
@@ -40,9 +47,51 @@ Here are some key features of YuniKorn.
 - Customized resource types (like GPU) scheduling support.
 - Rich placement constraints support.
 - Automatically map incoming container requests to queues by policies. 
-- Node partition: partition cluster to sub-clusters with dedicated quota/ACL management. 
+- Node partition: partition cluster to sub-clusters with dedicated quota/ACL management.
+
+### Integration with K8s:
+
+The `k8shim` provides the integration for K8s scheduling and supported features include: 
+
+- _Predicates:_ All kinds of predicates such as node-selector, pod affinity/anti-affinity, taint/tolerant, etc.
+- _Persistent volumes:_ We have verified hostpath, EBS, NFS, etc. 
+- _K8s namespace awareness:_ YuniKorn support hierarchical of queues, does it mean you need to give up K8s namespace? Answer is no, with simple config, YuniKorn is able to 
+ support automatically map K8s namespaces to YuniKorn queues. All K8s-namespace-related ResourceQuota, permissions will be still valid.
+- _Metrics:_ Prometheus, Grafana integration.
+- _Cluster AutoScaler_: YuniKorn can nicely work with Cluster AutoScaler (https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler) to drive cluster scales up and down.
+- _K8s Events_: YuniKorn also integrated with K8s events, so lots of information can be retrieved by using `kubectl describe pod`.
+
+#### Performance testing
+We love high-performance software, and we made tremendous efforts to make it to support large scale cluster and high-churning tasks. 
+Here's [Performance Test Result](docs/evaluate-perf-function-with-Kubemark.md) 
+
+#### Deployment model
+Yunikorn can be deployed as a K8s custom scheduler, and take over all POD scheduling. Community is actively working on 
+[Co-existing with other K8s schedulers](https://issues.apache.org/jira/browse/YUNIKORN-16) to allow YuniKorn take over subset of the cluster nodes. 
+ 
+#### Verified K8s versions 
+
+| K8s Version   | Support?  |
+| ------------- |:-------------:|
+| 1.12.x (or earlier) | X |
+| 1.13.x | √ |
+| 1.14.x | √ |
+| 1.15.x | √ |
+| 1.16.x | To be verified |
+| 1.17.x | To be verified |
 
-You can find more information about what are already supported and future plans in the [Road Map](docs/roadmap.md).
+### Web UI
+
+YuniKorn has builtin web UIs for queue hierarchies and apps. See below: 
+
+![Queues-UI](docs/images/screenshot-queues.png)
```

Comment:
UI images are not very high quality, but I guess you can leave it here.
I will try to grab some latest UI images and upload to here, in another PR.

Reply:
The quality is the same as the already existing ones. It is either the color sync used or the encoding that is causing the issue. Might need to grab them while being displayed on a different screen.

Reply:
follow up jira will add new screen shots